app/main/views/buyers.py

```diff
@@ -54,12 +56,17 @@ def update_brief(brief_id):
                                   'applications_closed_at': request.form['closed_at']
                                   },
                        'update_details': {'updated_by': current_user.email_address}}).get('briefs')
-        elif request.form.get('edit_seller_email_list'):
-            edit_seller_email_list = request.form['edit_seller_email_list'].split(get_new_line())
+        elif 'edit_seller_email_list' in request.form:
+            edit_seller_email_list = request.form.get('edit_seller_email_list', []).split(get_new_line())
```

Comment:
isn't the new line a constant?  you could do 
`NEW_LINE = '\n'`
at top of file rather than defining a function